.github/workflows/merge-pr.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   changes:
     name: Determine Changed Files
     runs-on: ubuntu-22.04
-    container: ghcr.io/libertydsnp/frequency/ci-base-image:1.0.0
+    container: ghcr.io/libertydsnp/frequency/ci-base-image:latest
```

Comment:
Using `latest` saves making these changes every time the ci-base-image changes, and allows ci-base-image to easily track versions.